src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <div class="container">
+        <label>
+          <img
+            class="icon"
+            src="images/Search.svg"
+            alt="Search"
+          >
+          <input
+            type="text"
+            id="bigSearchBar"
+            name="bigSearchBar"
+            class="form-field"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+    </form>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="small"
+    >
+      <div class="container">
+        <label>
+          <img
+            class="icon mini"
+            src="images/Search.svg"
+            alt="Search mini"
+          >
+          <input
+            type="text"
+            id="smallSearchBar"
+            name="smallSearchBar"
+            class="form-field small"
```

Comment:
```suggestion
            class="form-field form-field--small"
```

Rename classes-modifiers due to checklist
![image](https://user-images.githubusercontent.com/80700247/231773011-6591d326-0e01-44dd-b6e4-c0b08e3b086d.png)

Fix everywhere